.github/workflows/lint.yaml

```diff
@@ -74,13 +74,13 @@ jobs:
             version="${prefix}.$((minor + 1)).${suffix}"
             version="${version/-*-g*/}-dev"
           fi
-          echo "Required version for check: ${version}"
-          ./util/crs-rules-check/rules-check.py \
-            --output=github \
-            -r crs-setup.conf.example \
-            -r rules/*.conf \
-            -t util/APPROVED_TAGS \
-            "-v ${version}"
+          #echo "Required version for check: ${version}"
```

Comment:
I'm not sure it's a good idea to block this check completely.

Instead of this, we should comment only [this](https://github.com/coreruleset/coreruleset/blob/main/util/crs-rules-check/rules-check.py#L1103-L1113) block in linter itself, so only the `ver` action check will be suspended, not the whole check.

Reply:
No, I don't want to change the linter.

I want to release now. Then post release we set everything back as it was.

And we create an issue to follow up before the next release.